module/observations/helper.js

```diff
@@ -2115,4 +2117,280 @@ module.exports = class ObservationsHelper {
       }
     });
   }
+
+
+  static async createNewObservation(req){
```

Comment:
@borkarsaish65 we should add proper function doc, to new functions and existing functions (if it is missing)

Reply:
done